### PR TITLE
Browser binary location parameter

### PR DESCRIPTION
**Pull request by Brrovko**
- Add Environment Variables to configs
- Add browser binary location parameter
- Remove Navigate("about:blank") DriverService method.

### CODE REVIEW

**Review by Georgiy-Petrov-Quantori — DISMISSED** [review]

**Review by vladimir-s-quantory — APPROVED** [review]

File: src/Behavioral.Automation/Services/DriverService.cs [hunk 1]
```diff
@@ -25,7 +25,6 @@ public sealed class DriverService : IDriverService
         public DriverService([NotNull] IScopeContextManager scopeContextManager)
         {
             _scopeContextManager = scopeContextManager;
-            Navigate("about:blank");
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Services/DriverService.cs:28]: I think this line was added intentionally by @SergeyRuzin, not sure if it should be removed.

Comment (Brrovko) [reply src/Behavioral.Automation/Services/DriverService.cs:28]: Don't worry I've confirmed with Sergey.

Comment (SergeyRuzin) [reply src/Behavioral.Automation/Services/DriverService.cs:28]: I'm not sure it was added by me. Still we should check that it doesn't break anything

**Review by SergeyRuzin — APPROVED** [review]

File: src/Behavioral.Automation/Services/DriverService.cs (hunk 1 above)
Comment (Brrovko) [reply src/Behavioral.Automation/Services/DriverService.cs:28]: I've checked. Working fine. The reason why I removed is that not possible to test desktop applications based on electronjs. The page becomes blank.

**Review by afomina-quantori — COMMENTED** [review]
I afraid it may lead to problems in case of registration of multiple pages with different scopes. Should be tested probably on Glass project.

File: src/Behavioral.Automation/Services/ConfigServiceBase.cs [hunk 2]
```diff
@@ -8,7 +8,10 @@ namespace Behavioral.Automation.Services
     public static class ConfigServiceBase
     {
         public static readonly IConfigurationRoot ConfigRoot =
-            new ConfigurationBuilder().AddJsonFile("AutomationConfig.json").Build();
+            new ConfigurationBuilder()
+                .AddJsonFile("AutomationConfig.json")
+                .AddEnvironmentVariables()
```
[the remark this answers is not in the document]
Comment (Brrovko) [reply src/Behavioral.Automation/Services/ConfigServiceBase.cs:13]: Added to the README.md

**Review by vladimir-s-quantory — APPROVED** [review]

**Review by SergeyRuzin — APPROVED** [review]

**Review by Georgiy-Petrov-Quantori — APPROVED** [review]

**Pull request merged**